### ISSUE TITLE
Controls disappeared on Foundry 11.299

### ISSUE THREAD

**Issue by mathsteck** [bekriebel/fvtt-module-avclient-livekit#53]
The controllers for LiveKit video and audio disappeared on v11.299.

**Comment by bekriebel** [comment]
Thank you for letting me know! I haven't had the chance to test with the latest version of v11 yet. I'll try to get an update out later this week.

**Comment by mathsteck** [comment]
It looks like this:
![image](https://github.com/bekriebel/fvtt-module-avclient-livekit/assets/3390366/85c4f400-6e3f-4cff-b624-3352d6b91212)
It usually have some controls at the bottom that allows you to reconnect, enable disable camera... But as you can see they disappeared.

I have no modules installed (besides LiveKit) and I'm using the Age of Sigmar System. I tested in both Firefox and Chrome and didn't really matter, both of them don't have the controls enabled. I can try using another system and see if that's also happening.

**Comment by mathsteck** [comment]
I tried with other clean systems/worlds and the issue is still present.

**Comment by bekriebel** [comment]
@mathsteck Ahh, thanks for the extra info. I tried with the Age of Sigmar system, and I can recreate the issue. However, the issue also exists with the LiveKit module disabled. So, it appears that the issue is between core foundry and some of the systems. If you try with D&D 5e, everything works as expected.

I tested with Simple World Building and the native WebRTC, and it's also broken there, so I think this may be a core foundry bug.

I'll see if I can work around it with the module, but the issue should also be opened against Foundry since it happens with no modules at all enabled.
